### PR TITLE
OCPBUGSM-26455 - Change misleading message indicates we can support more than 3 masters

### PR DESCRIPTION
**Pull request by eliorerz**
message: "Clusters with less than 3 dedicated masters or a single worker are not supported" suggests that we support more than 3

Any attempt setting less (or more) than 3 masters - then the following warning is shown:
Warning alert: `Cluster is not ready to be installed yet
The following requirements must be met:
Clusters with less than 3 dedicated masters or a single worker are not supported. Please either add hosts, or disable the worker host`
The message should better indicate that we only support 3 masters, i.e. not less and not more.

https://bugzilla.redhat.com/show_bug.cgi?id=1939225

### CODE REVIEW

File: internal/operators/ocs/validation_test.go [hunk 1]
```diff
@@ -505,7 +505,7 @@ var _ = Describe("Ocs Operator use-cases", func() {
 				clust.AllHostsAreReadyToInstall:           {status: clust.ValidationSuccess, messagePattern: "All hosts in the cluster are ready to install"},
 				clust.IsDNSDomainDefined:                  {status: clust.ValidationSuccess, messagePattern: "The base domain is defined"},
 				clust.IsPullSecretSet:                     {status: clust.ValidationSuccess, messagePattern: "The pull secret is set"},
-				clust.SufficientMastersCount:              {status: clust.ValidationFailure, messagePattern: "Clusters with less than 3 dedicated masters or a single worker are not supported. Please either add hosts, or disable the worker host"},
+				clust.SufficientMastersCount:              {status: clust.ValidationFailure, messagePattern: "Clusters can only have 3 dedicated masters. Please either add hosts, or disable the worker host"},
```
Comment (osherdp) [inline internal/operators/ocs/validation_test.go:508]: find a way to include a suitable message to single-node
Maybe we can make it depend on the availability_mode parameter, as single-node is with 'None' and regular clusters are with 'Full'

**Review by osherdp — CHANGES_REQUESTED** [review]

File: internal/operators/ocs/validation_test.go (hunk 1 above)
Comment (eliorerz) [reply internal/operators/ocs/validation_test.go:508]: There is a single node message [here](https://github.com/openshift/assisted-service/blob/913aba63aae8923400e8ff4e040c8cfc0d59897e/internal/cluster/validator.go#L369)

File: internal/cluster/validator.go [hunk 2]
```diff
@@ -368,7 +368,7 @@ func (v *clusterValidator) printSufficientMastersCount(context *clusterPreproces
 		if noneHAMode {
 			return "Single-node clusters must have a single master node and no workers."
 		}
-		return fmt.Sprintf("Clusters with less than %d dedicated masters or a single worker are not supported. Please either add hosts, or disable the worker host",
+		return fmt.Sprintf("Clusters can only have %d dedicated masters. Please either add hosts, or disable the worker host",
```
Comment (osherdp) [inline internal/cluster/validator.go:371]: sorry for the late message
Maybe we should change it to ``"Clusters must have exactly %d dedicated masters"``?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**